tests/integration/test_file_editor.py

```diff
@@ -57,3 +61,50 @@ def test_file_editor_happy_path(temp_file):
     with open(temp_file, 'r') as f:
         content = f.read()
     assert 'This is a sample file.' in content
+
+
+def test_file_editor_with_xml_tag_parsing(temp_file):
```

Comment:
Can we copy the exact observation i got from https://github.com/All-Hands-AI/OpenHands/issues/5479 and include them in the test?

Reply:
In your trajectory I think the error happened when the agent 'viewed' the file `action_execution_server.py` and the parsing stopped right before the closing tag `</oh_aci_output>`.

```
Invalid JSON in 'openhands-aci' output: 
224\t            matches = re.findall(\n   225\t                r'<oh_aci_output>(.*?)
Invalid JSON in 'openhands-aci' output: (.*?)
```

I think this current file content makes more sense here because it already simulate what happened in your trajectory? Eventually we'll also update `action_execution_server.py` content with this parsing code.